### DISCUSSION TITLE
Does poetry work as usual with `virtual.in-project = true`?

### DISCUSSION THREAD

**Discussion by albas99** [python-poetry/poetry#7719] [Q&A]
I'm new to Poetry and had some issues with the Zed editor seeing my environment so I changed my configuration to create the virtual environment in the root folder of my projects, but I noticed it's saving it in a folder called `.venv`. My question is does poetry still work as normal and recognizes this folder/environment as its and handles dependencies as usual(when it comes to conflicts and versioning) or is this new environment a generic python venv created by poetry under the hood?

**Comment by dimbleby** [comment]
it's not clear what you're asking.  Do you have in mind a particular thing that you are worried that poetry will or will not do?

If yes, then you can just test it and answer your own question.  If no... then it's not clear what you're asking!
